### DISCUSSION TITLE
Searching for the config in the screenshot

### DISCUSSION THREAD

**Discussion by bebehei** [ldelossa/nvim-ide#120] [Q&A]
Hi all,

I'm currently searching for the configuration of the screenshot, which is referenced in README.md. Can somebody help?

It's this screenshot:

![](https://github.com/ldelossa/nvim-ide/raw/95c2ca7286b65642140077fa201b2c53de746714/contrib/screenshot.png)

Where can I get the config from?

**Comment by bebehei** [comment]
In the meantime, I've found @ldelossa's dotfiles and a [promisingly named configuration file](https://github.com/ldelossa/dotfiles/blob/master/config/nvim/lua/configs/nvim-ide.lua). But unfortunately, this does not help in any way.

**Reply by ldelossa** [reply]
I'm confused, the entire config is defined in the readme and also in vim help. Is something missing?

**Reply by ldelossa** [reply]
P.s. im a big fan of dunst 😀

**Reply by bebehei** [reply]
Thanks. I had been mainly struggling with the the terminal. How do I open it?

**Reply by ldelossa** [reply]
https://github.com/ldelossa/nvim-ide/blob/cf1ed20208750edc1b26ee187c1724aa78da8243/doc/nvim-ide.txt#L654

Check this section.
